src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/UrisAreRelative_Expected.sarif

```diff
@@ -17,10 +17,15 @@
         }
       },
       "results": [],
+      "files": {
+        "file:///C:/Code/sarif-sdk/src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/UrisAreRelative.sarif": {
+          "mimeType": "application/sarif-json"
+        }
+      },
```

Comment:
>} [](start = 6, length = 1)

All the other "expected" files already had a `run.files` property; I guess this one hadn't been regenerated in a while.